pyproject.toml

```diff
@@ -44,7 +44,7 @@ chardet = "3.0.4"
 "collective.recipe.modwsgi" = "2.1"
 "collective.recipe.template" = "1.13"
 colorama = "0.3.3"
-dcicsnovault = ">=3.0.0,<4"
+dcicsnovault = ">=3.0.2,<4"
```

Comment:
Probably only re-lock was needed but given you were already in there, this can't hurt.

Reply:
My point here was to make `poetry install` fail with snovault prior to this fix (don't want any prior version 3 installs).